src/main/java/org/wecancodeit/pantryplus/CountedLineItem.java

```diff
@@ -0,0 +1,61 @@
+package org.wecancodeit.pantryplus;
+
+import javax.persistence.Entity;
+
+@Entity
+public class CountedLineItem extends LineItem {
+
+	protected int quantity;
+
+	public CountedLineItem() {
+
+	}
+
+	public CountedLineItem(Cart cart, Product product, int quantity) {
+		this.cart = cart;
+		this.product = product;
```

Comment:
Should call super(cart, product) instead of setting the fields for cart and product.